docs/getting_started.txt

```diff
@@ -58,9 +58,96 @@ Entry Points for Automation
 So you have an application, you know it supports one of the mentioned 
 accessibility technologies. What's the next?
 
-**Application** and **Desktop** objects are backend-specific.
+First you should start your application or connect to the existing app
```

Comment:
...connect to an existing application ... 
(We didn't mention any application before so it's still indefinite)